axelrod/game.py

```diff
@@ -110,10 +139,19 @@ def __init__(self, r: Score = 3, s: Score = 0, t: Score = 5, p: Score = 1) -> No
             Score obtained by a player for defecting against a cooperator.
         p: int or float
             Score obtained by both player for mutual defection.
+        **attributes
+            optional attributes for the game. Used
+            to ensure strategies used with the game are valid.
+
         """
         A = np.array([[r, s], [t, p]])
 
-        super().__init__(A, A.transpose())
+        default_attributes = {
+            'has_RPST': True
```

Comment:
This could be explicitly tested for, either with a try-catch blog or by inspecting the class for the presence of an RPST() function.

That said, is RPST synonymous with the game being IPD?